ci/ray_ci/BUILD.bazel

```diff
@@ -48,6 +48,39 @@ py_binary(
     deps = [":ray_ci_lib"],
 )
 
+py_binary(
+    name = "bisect_test",
+    srcs = ["bisect_test.py"],
+    exec_compatible_with = ["//:hermetic_python"],
+    deps = [":ray_ci_lib"],
+    data = [":macos_bisect_script"],
+)
+
+genrule(
+    name = "macos_bisect_script",
+    srcs = ["macos/macos_ci_test.sh"],
+    outs = ["macos_ci_test.sh"],
+    cmd = """
+        cat $(location macos/macos_ci_test.sh) > $@
+    """,
+)
```

Comment:
runfiles create a symlink to this file to the repository that keep changing (because of bisection); so i want to have keep a constant hard copy in runfiles